src/components/AnnotationPopover/AnnotationPopover.js

```diff
@@ -79,19 +85,19 @@ class AnnotationPopover extends React.PureComponent<Props> {
             <Internationalize language={language} messages={intlMessages}>
                 <div
                     className={classNames(CLASS_ANNOTATION_POPOVER, {
-                        'ba-inline': isInline,
-                        'ba-animate-popover': isMobile,
-                        'ba-create-popover': isPending
+                        CLASS_INLINE_POPOVER: isInline,
```

Comment:
Since these are keys in an object, you'll need to enclose them in square braces for their value to be used. 

ie)
```
{
  [CLASS_INLINE_POPOVER]: isInline,
  // etc.
}
```